charts/sftpgo/values.yaml

```diff
@@ -163,6 +163,57 @@ services: {}
   #       nodePort:
   #   externalTrafficPolicy:
 
+ingress:
+  # Ingress for the Web UIs (user and admin interface)
+  app:
+    # -- Create an Ingress for the user and admin Web UI
+    enabled: false
+    # -- Ingress class name. See [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for details.
+    className: ""
+    # -- Annotations to be added to the Ingress.
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+
+    # -- List of virtual hosts. See [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for details.
+    hosts: []
+    #  - host: chart-example.local
+    #    # Change httpd.web_root in the sftpgo configuration if you change the path
+    #    paths:
+    #      - path: /
+    #        pathType: ImplementationSpecific
+
+    # -- List of TLS secrets. See [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for details.
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+  # Ingress for the API endpoint
+  api:
+    # -- Create an Ingress for the API and health endpoint.
+    enabled: false
+    # -- Ingress class name. See [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for details.
+    className: ""
+    # -- Annotations to be added to the Ingress.
+    annotations: {}
+      # nginx.ingress.kubernetes.io/rewrite-target: /$1
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+
+    # -- List of virtual hosts. See [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for details.
+    hosts: []
+    #  - host: chart-example.local
+    #    paths:
+    #      - path: /(api/.*|healthz)$
```

Comment:
Is there a reason why healthz should be included here?

Reply:
sftpgo applies `httpd.web_root` to the UI only. So, when changing the UI to `/my/base/path`, the health check and API endpoints are still located in the root path. Probably this is more obvious when also looking at the rewrite annotation:

```yaml
nginx.ingress.kubernetes.io/rewrite-target: /$1
```

Reply:
And more specifically, the health endpoint is usefully in my Ansible roles to wait for sftpgo to become available.

Reply:
I see. I think people normally shouldn't expose the health endpoint. I get that it's useful in your case, but I'd rather remove it from the default, because people tend to go with that without thinking.

Reply:
Do you consider the health endpoint harmful?

Reply:
I've seen quite a few cases where the health endpoint was essentially a DoS endpoint as well, so as a rule of thumb I usually prefer keeping it closed from the public. That's also what the least privilege principle says.